<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIDataTypeList</class>
 <widget class="QWidget" name="SIDataTypeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxDetails">
     <property name="title">
      <string>Data Type List:</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>21</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>2</y>
         <width>341</width>
         <height>38</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolAddType">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Create and add new data type.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Add Data Type</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolDeleteType">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Delete active data type.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Delete Data Type</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolFieldAdd">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Add new field</string>
          </property>
          <property name="toolTipDuration">
           <number>-4</number>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Add Field</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolFieldDelete">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Insert new field.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Delete Field</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolFieldInsert">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Remove selected field</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Insert Field</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolMoveUp">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Move selection up.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Move Up</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolMoveDown">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Move selection down.</string>
          </property>
          <property name="text">
           <string>.....</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/Move Down</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>511</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeTypes">
         <property name="cursor" stdset="0">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed|QAbstractItemView::EditTrigger::SelectedClicked</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>50</number>
         </attribute>
         <column>
          <property name="text">
           <string>Name:</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data Type:</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default Value:</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
